src/overlays/actors/ovl_En_Ru2/z_en_ru2.c

```diff
@@ -111,41 +140,41 @@ void EnRu2_Destroy(Actor* thisx, PlayState* play) {
     Collider_DestroyCylinder(play, &this->collider);
 }
 
-void func_80AF2608(EnRu2* this) {
+void EnRu2_UpdateEyes(EnRu2* this) {
     s32 pad[3];
-    s16* unk_2A6 = &this->unk_2A6;
-    s16* unk_2A4 = &this->unk_2A4;
+    s16* eyeTextureFrame = &this->eyeTextureFrame;
+    s16* eyeTextureIndex = &this->eyeTextureIndex;
 
-    if (DECR(*unk_2A6) == 0) {
-        *unk_2A6 = Rand_S16Offset(0x3C, 0x3C);
+    if (DECR(*eyeTextureFrame) == 0) {
+        *eyeTextureFrame = Rand_S16Offset(60, 60);
     }
 
-    *unk_2A4 = *unk_2A6;
-    if (*unk_2A4 >= 3) {
-        *unk_2A4 = 0;
+    *eyeTextureIndex = *eyeTextureFrame;
+    if (*eyeTextureIndex >= 3) {
+        *eyeTextureIndex = 0;
     }
 }
 
-s32 func_80AF2690(EnRu2* this) {
+s32 EnRu2_GetSwitchFlag(EnRu2* this) {
     s32 params_shift = PARAMS_GET_U(this->actor.params, 8, 8);
 
     return params_shift;
 }
 
-s32 func_80AF26A0(EnRu2* this) {
+s32 EnRu2_GetType(EnRu2* this) {
     s32 params = PARAMS_GET_U(this->actor.params, 0, 8);
 
     return params;
 }
 
 #if OOT_DEBUG
 void func_80AF26AC(EnRu2* this) {
-    this->action = 7;
-    this->drawConfig = 0;
+    this->action = ENRU2_CROSSING_ARMS_INVISIBLE;
+    this->drawConfig = ENRU2_DRAW_NOTHING;
     this->alpha = 0;
-    this->unk_2B8 = 0;
+    this->isLightBall = 0;
```

Comment:
`this->isLightBall = false`? (and `if (!this->isLightBall)` and `this->isLightBall = true` below)